src/background.js

```diff
@@ -24,8 +24,8 @@ chrome.pageAction.onClicked.addListener(function(){
 });
 
 chrome.tabs.onUpdated.addListener(function(tabId, changeInfo, tab){
-  const regexUrl = /https:\/\/github\.com\/.*\/.*\/compare\/.*/;
-  if(tab.url.match(regexUrl)){
+  const regexUrl = /https:\/\/github\.com\/.*\/.*\/compare\/..*/;
+  if(regexUrl.test(tab.url) && changeInfo.url){
```

Comment:
There are a coupe of pages that you go through to fully navigate to the actual PR page, and those pages trigger their own loading events with urls. Since we are injecting the content script early (also a fix for pushState issues) the fill-pull-body will receive those early messages and start recursively looking for the pull_request_body element.